srcpkgs/dhcpcd/template

```diff
@@ -11,22 +11,21 @@ maintainer="Juan RP <xtraeme@voidlinux.org>"
 license="BSD-2-Clause"
 homepage="https://roy.marples.name/projects/dhcpcd"
 distfiles="https://roy.marples.name/downloads/dhcpcd/dhcpcd-${version}.tar.xz"
-checksum=96968e883369ab4afd11eba9dfd9bb109f5dfff65b2814ce6c432f36362dc9b5
+checksum=019ff03f4495c58be1b43a5543a4f427b6a2c3f6a3850a112bc0baca50771243
 lib32disabled=yes
 conf_files=/etc/dhcpcd.conf
 
 pre_install() {
 	# make wpa_supplicant log to syslog rather than stdout
```

Comment:
Why is this hook still being used?
Update to wpa_supplicant-2.6 or newer with CONFIG_MATCH_IFACE enabled and then simply add -M to the start arguments for it. Voila, exactly the same as this hook with the benefit of being more reliable.